megamek/src/megamek/common/WeaponType.java

```diff
@@ -411,11 +411,11 @@ public int getFireTN() {
         } else if (hasFlag(F_FLAMER)) {
             return 4;
         } else if (hasFlag(F_PLASMA)) {
-            return 2;
+            return TargetRoll.AUTOMATIC_SUCCESS;
         } else if (hasFlag(F_PLASMA_MFUK)) {
```

Comment:
I know what it actually stands for, but I get a good chuckle every time I see this acronym